content/en/golden_signals.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Golden Signals
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+[Golden signals](https://sre.google/sre-book/monitoring-distributed-systems/#xref_monitoring_golden-signals) are metrics that help us understand how [services](https://glossary.cncf.io/service/) are experienced by users. Today, it is the gold standard for monitoring application performance and focuses on four specific metrics: Latency, traffic, errors, and saturation. The first three metrics relate to requests. A request is a message sent from one service to another. Services do that to collaborate and work as a [distributed app](https://glossary.cncf.io/distributed_apps/).
+
+*Latency* is the time it takes to service a request. *Traffic* measures the request frequency placed on the application. *Errors* measure the request failure rate. And *Saturation* measures the capacity of app resources, allowing users to address bottlenecks before they become a problem.
```

Comment:
We really want to make sure the glossary is consistent in style, so instead of subheadings, I added them all in a paragraph. I also removed the details after errors as that was too much detail for this section

Reply:
Ok, great! In that case, let's end with "distributed app."  Rest looks good to you, @iamNoah1?